ArduPlane/system.cpp

```diff
@@ -7,11 +7,60 @@
 *
 *****************************************************************************/
 
+bool validate_fence_frame(float newframe)
+{
+    switch ((int)newframe) {
+        case 0:
+            ::fprintf(stderr, "Requested frame: MSL\n");
+            break;
+        case 1:
+            ::fprintf(stderr, "Requested frame: REL/HOME\n");
+            break;
+        case 2:
+            ::fprintf(stderr, "Requested frame: ORIGIN\n");
```

Comment:
let's not provide the option to set the fence alt above ekf origin.  just msl, origin and terrain for now would be good.